gnocchi/rest/api.py

```diff
@@ -188,6 +188,20 @@ def get_header_option(name, params):
                      options.get(name, params.get(name, 'false')))
 
 
+def get_header_option_array(name, params):
+    type, options = werkzeug.http.parse_options_header(
+        pecan.request.headers.get('Accept'))
```

Comment:
I don't see a Gabbi test for the `Accept` case – i'm not even sure I understand how it works. Can you add one?

Reply:
I've added a test for `Accept` case.